urlchecker/core/urlproc.py

```diff
@@ -30,16 +30,19 @@ def check_response_status_code(url, response):
     """
     # Case 1: response is None indicating triggered error
     if not response:
-        print_failure(url)
+        if print_level not in ["success-only", "none"] :
```

Comment:
What about files-with-urls-only?

Reply:
I don't need it here, because I want to show failures when `files-with-urls-only` is specified.